docs/src/architecture/mapper.md

```diff
@@ -160,7 +160,7 @@ sudo systemctl restart tedge-mapper-az.service
 
 The AWS mapper takes messages formatted in the [Thin Edge JSON](thin-edge-json.md) as input.
 It validates if the incoming message is correctly formatted Thin Edge JSON, then outputs the message.
-The validated messages are published on the topic `aws/messages/` from where they are forwarded to AWS.
+The validated messages are published on the topic `aws/td/` from where they are forwarded to AWS.
```

Comment:
is `aws/td/` a valid topic?, Because there is a forward slash at the end.

Reply:
Right, that should probably say `aws/td/#`.